variables.tf

```diff
@@ -124,3 +124,23 @@ variable "stop_timeout" {
   }
   default = 30
 }
+
+
+variable "deployment_circuit_breaker" {
+  description = "(Optional) Configuration block for deployment circuit breaker"
+  type = object({
+    enable   = optional(bool, true)
+    rollback = optional(bool, true)
+  })
+  default = {}
+}
+
+variable "provider_timeouts" {
```

Comment:
@rbapst-tamedia what about `service_timeouts`?

Reply:
yes because it's for the `aws_ecs_service`resource. Then why not `aws_ecs_service_timeout` ?

Reply:
For the circuit breaker we are not using the prefix `aws_ecs_`, therefore i would stick to just `service_timeouts`, but I let you decide. Maybe we can use the plurals (timeouts).

Reply:
INFO: circuit_breaker is only for aws_ecs_service. Timeouts is a "reserved" variable for the provider. So we could have timeouts for aws_ec2_instances, aws_xyz...
But I agree, lets go with your proposition: `service_timeouts`.